Tests/AWSSDKSwiftCoreTests/AWSClientTests.swift

```diff
@@ -406,6 +406,35 @@ class AWSClientTests: XCTestCase {
             XCTFail(error.localizedDescription)
         }
     }
+    
+    func testSignedClient() {
+        let input = E()
+        let client = AWSClient(
+            accessKeyId: "foo",
+            secretAccessKey: "bar",
+            region: .useast1,
+            service: "s3",
+            serviceProtocol: .restxml,
+            apiVersion: "2013-12-02",
+            middlewares: [],
+            httpClientProvider: .createNew
+        )
+
+        do {
+            let awsRequest = try client.createAWSRequest(
```

Comment:
Can you try this with different requests, maybe a GET request as well, seeing as that was originally the requests that used SignedURL.

Reply:
good point. would you like separate tests for that or is it ok if I test multiple requests in a single test?

Reply:
It's ok to put both of those in the same test